### PR TITLE
update crossword beta link

### PR DESCRIPTION
**Pull request by NathanielBennett**
Updates the text and target of the crosswords optout link - so it's no longer an optout link and points to the latest crossword blog ..

### CODE REVIEW

File: common/app/views/fragments/crosswordsOptOutBanner.scala.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <div class="site-message--crosswords" data-link-name="release message" role="dialog" aria-label="welcome" aria-describedby="site-message__message">
         <div class="gs-container">
             <div class="u-cf">
-                You're viewing a new beta version of the Guardian's crosswords. To return to the current version, <a href="@LinkTo("/crosswords/optout")">click here.</a>
+                You're viewing a new version of the Guardian's crosswords.<a href="@LinkTo("/crosswords/crossword-blog/2015/sep/11/beta-testing-new-guardian-crossword-site")">Read more here..</a>
```
Comment (rich-nguyen) [inline common/app/views/fragments/crosswordsOptOutBanner.scala.html]: Should that be `Read more here.` ?

Comment (rich-nguyen) [inline common/app/views/fragments/crosswordsOptOutBanner.scala.html]: And is there a whitespace between `.<a`

**Comment by rich-nguyen** [comment]
:+1:

File: common/app/views/fragments/crosswordsOptOutBanner.scala.html (hunk 1 above)
Comment (NathanielBennett) [inline common/app/views/fragments/crosswordsOptOutBanner.scala.html]: Yes. Thanks @rich-nguyen

**Pull request merged**